src/resources/views/components/modals/following.blade.php

```diff
@@ -4,32 +4,53 @@ class="fixed left-0 top-0 bg-mydark bg-opacity-50 w-full h-full justify-center i
         class="bg-mycream rounded-lg shadow-md p-10 flex justify-center items-center">
         <div id="followingModal" class="bg-mycream mx-auto overflow-hidden transition-opacity duration-500">
             <div class="text-mydark font-bold text-center w-full mb-2">
-                <h1>People you followed</h1>
+                <h1>Followed Users</h1>
             </div>
+
+            @php
+                $followedUserIds = $user->followedUsers()->where('status', 1)->pluck('following_id')->toArray();
+                $followedUsers = App\Models\User::whereIn('id', array_diff($followedUserIds,[$user->id]))->get();
+            @endphp
```

Comment:
I noticed that you always use explicit variables like this. Much better to use mutations for this and then $append = [] so that if you ever need to call that user(model), you can use the appended methods that return some data based on your desired results.

Reply:
will use $appends, sir